docs/data/product/dea-waterbodies-landsat/_data.yaml

```diff
@@ -15,8 +15,10 @@ spatial_data_type: Vector
 time_span:
   start: 1986
   end: Present
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
```

Comment:
I _think_ this is updated more like weekly - @BexDunn?

Reply:
Thanks - I talked to Bex who said it is Daily.